src/containers/Prizes.js

```diff
@@ -1,17 +1,122 @@
-import React from 'react';
-import styled from 'styled-components';
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { db } from '../utility/firebase'
+import { DB_COLLECTION, DB_HACKATHON } from '../utility/Constants'
+import { CardLike, DetailContainer, DetailColumn, chunkify } from '../components/Common'
+import { H2, UL, LI, I } from '../components/Typography';
 
 const CenteredH1 = styled.h1`
   text-align: center;
 `
+const COLUMNS_OF_PRIZES = 3;
 
+const PrizeCard = styled.div`
+  ${CardLike};
+  margin: 0.5em 0;
+`
 
-export default () => {
+const getPrizes = () => {
+  return db
+    .collection(DB_COLLECTION)
+    .doc(DB_HACKATHON)
+    .collection('Prizes')
+    .orderBy('place', 'asc')
+    .get()
+    .then(querySnapshot => {
+      return querySnapshot.docs
+    });
+}
+
+const createPrizeList = (prizes) => {
+  return (
+    <DetailContainer>
+      {
+        chunkify(prizes, COLUMNS_OF_PRIZES, true).map((chunk, i) =>
+          <DetailColumn key={i}>{chunk.map(singlePrize)}</DetailColumn>
+        )
+      }
+    </DetailContainer>
+  );
+}
+
+
+const singlePrize = (prize) => {
+  return (
+    <PrizeCard>
+      <H2>{prize.title}</H2>
+      { (prize.sponsor === '') ? null : <I>{`Sponsored by ${prize.sponsor}`}</I>}
+      <UL>
+        {prize.content.split("|").map((item) => <LI key={prize.title + item}>{item}</LI>)}
+      </UL>
+    </PrizeCard>
+  );
+}
 
+
+export const MainPrizes = () => {
+  const [mainPrizes, setMainPrizes] = useState([]);
```

Comment:
not sure if this would make a diff, but maybe better to maintain prize state in one container component and filter main + sponsor prizes to pass to 2 diff presentational components? That way we wouldn't have to make 2 `getPrizes` db calls every render

Reply:
sounds goodo i'll try this change